pkg/apis/core/v1/helper/helpers.go

```diff
@@ -495,19 +494,6 @@ func GetMatchingTolerations(taints []v1.Taint, tolerations []v1.Toleration) (boo
 	return true, result
 }
 
-// GetAvoidPodsFromNodeAnnotations scans the list of annotations and
-// returns the pods that needs to be avoided for this node from scheduling
-func GetAvoidPodsFromNodeAnnotations(annotations map[string]string) (v1.AvoidPods, error) {
```

Comment:
I guess you're planning to move the rest of the things after this initial PR is merged?

Reply:
Yeah, this is an initial helper so the repository is not empty. The relevant rest will follow right after.